docs/README.md

```diff
@@ -387,6 +387,14 @@ Loads the content of the last commit message that triggered the action into `COM
       - uses: Alfresco/alfresco-build-tools/.github/actions/get-commit-message@ref
 ```
 
+To use the action in a proper way, be sure to use a checkout with fetch-depth option as follow:
```

Comment:
```suggestion
This action requires a checkout with fetch-depth option as follow:
```

I would merge the two snippet in a single one (beware of missing dash in the checkout step)